spacy/pipeline/functions.py

```diff
@@ -16,7 +16,8 @@ def merge_noun_chunks(doc):
         return doc
     with doc.retokenize() as retokenizer:
         for np in doc.noun_chunks:
-            attrs = {"tag": np.root.tag, "dep": np.root.dep}
+            lemma = doc[np.start:np.end].lemma_
+            attrs = {"tag": np.root.tag, "dep": np.root.dep, "lemma": lemma, "ent_type": np.root.ent_type}
             retokenizer.merge(np, attrs=attrs)
```

Comment:
Is this behaviour specific to the `merge_noun_chuncks` function, or should this be fixed in the underlying `retokenizer.merge` functionality?

Reply:
@svlandeg I do think we want to take care of this in the retokenizer. Noun chunks are always going to be a subtree, but for other merges that work as a subtree it's good to keep the parse working properly. I actually thought we already did this?